submissions/zhenyakornilov/js-dom/css/styles.css

```diff
@@ -0,0 +1,422 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+ul,
```

Comment:
this style applies just for `li` tags, so you can leave just one tag here
https://developer.mozilla.org/en-US/docs/Web/CSS/list-style